tasks/aiida-daemon.yml

```diff
@@ -18,6 +18,14 @@
     daemon_user: "{{ current_user }}"
     daemon_user_home: "{{ current_user_home }}"
     daemon_aiida_venv: "{{ aiida_venv |  regex_replace('\\$\\{HOME}', current_user_home) }}"
+  when: ansible_service_mgr == "systemd"
+
+- name: Check if the AiiDA daemon is running
+  shell: pgrep -a verdi | grep run_daemon | awk '{print $1}'
+  changed_when: false
+  register: aiida_daemon_status
+  failed_when: aiida_daemon_status.rc not in [0,1]
+  when: ansible_service_mgr != "systemd"
```

Comment:
How about looking for `not running` in the output of `verdi daemon status`?

Reply:
That we could do, but would have to change it. Now it is fairly general unless we change the name of the process. What do you think?